webapp/src/components/Bid/Bid.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { Grid, Button } from 'semantic-ui-react'
+import { t } from '@dapps/modules/translation/utils'
+
+import { locations } from 'locations'
+import { bidType } from 'components/types'
+import { distanceInWordsToNow, shortenAddress } from 'lib/utils'
+import AddressBlock from 'components/AddressBlock'
+import Mana from 'components/Mana'
+
+import './Bid.css'
+
+export default class Bid extends React.PureComponent {
+  static propTypes = {
+    isOwner: PropTypes.bool.isRequired,
+    bid: bidType.isRequired
+  }
+
+  render() {
+    const { bid, isOwner, onConfirm, onUpdate } = this.props
+
+    return (
+      <Grid stackable className={'Bid'}>
+        <Grid.Row>
+          <Grid.Column width={isOwner ? 4 : 2}>
+            <h3>{t('global.from')}</h3>
+            <div className="address-wrapper" title={bid.bidder}>
+              <Link to={locations.profilePageDefault(bid.bidder)}>
+                <AddressBlock
+                  address={bid.bidder}
+                  scale={3.4}
+                  hasTooltip={false}
+                  hasLink={false}
+                />&nbsp;
+                <span className={`short-address ${!isOwner ? 'you' : ''}`}>
+                  {isOwner ? shortenAddress(bid.bidder) : t('global.you')}
```

Comment:
I think it's a bit weird that the same condition is used in two different ways here, `isOwner ?` and `!isOwner ?`. Maybe we could use `isOwner ?` both times?